sdks/java/io/iceberg/src/main/java/org/apache/beam/sdk/io/iceberg/IcebergWriteResult.java

```diff
@@ -15,7 +15,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.apache.beam.io.iceberg;
+package org.apache.beam.sdk.io.iceberg;
```

Comment:
What is the PCoder used for the `PCollection<KV<String, Snapshot>>` here? I'm concerned there may be update compatibility issues with it.

Reply:
Looks like it's SerializableCoder:
https://github.com/apache/beam/blob/d05196dea8a393ef2afec8e95c49a309375a42d9/sdks/java/io/iceberg/src/main/java/org/apache/beam/io/iceberg/AppendFilesToTables.java#L63-L66

Reply:
That'll be dangerous. Let's see if we can remove this. (Fortunately we don't have any existing Iceberg to stay compatible with.)

Reply:
> Let's see if we can remove this

Do you mean we should remove the Snapshot part? e.g. only output the table string identifier instead of a KV<table identifier, Snapshot> ?

Reply:
We should probably output a Row (or, rather schema'd POJO) with table identifier and any relevant information from snapshot (which can be safely expanded in the future if we use schema'd data).

Reply:
Sounds good, I created a new AutoValue class (SnapshotInfo) to represent this data. Lmk if this is what you had in mind